tests/test_sphinx/test_build_no_ext.resolved.xml

```diff
@@ -26,17 +26,21 @@
                         <paragraph>
                             <inline ids="id6">
                                 [
-                                <citation_reference docname="part1/chap2" refname="id5">
```

Comment:
@AakashGfude I am most concerned by this change. It seems to be not resolving a `citation_reference`